applications/smart-distancing/libs/config_engine.py

```diff
@@ -2,8 +2,17 @@
 import configparser
 import threading
 
+
 class ConfigEngine:
-    def __init__(self, config_path = './config-skeleton.ini'):
+    """
+    Handle the .ini confige file and provide a convenient interface to read the parameters from config.
+    When an instance of ConfigeEngine is created you can use or pass it to other classes/ modules in
+    which you want to access the parameters at config file
```

Comment:
Correction: " When an instance of ConfigeEngine is created you can use/pass it to other classes/modules that needs access to the parameters at config file. "